composer.json

```diff
@@ -5,7 +5,7 @@
     "homepage": "https://github.com/studiopress/genesis-custom-blocks",
     "license": "GPL-2.0",
     "require": {
-        "php": "^5.6 || ^7 || ^8"
+        "php": "^7 || ^8"
```

Comment:
Core `6.3` [doesn't support](https://make.wordpress.org/core/2023/07/05/dropping-support-for-php-5/) PHP `5.6` anymore. 

That's a good excuse to remove our support, as it's really hard to make `composer install` work for PHP `5.6`-`8.2`.